abci/tutorials/abci-v2-forum-app/README.md

```diff
@@ -0,0 +1,28 @@
+## Forum Application
+
+The **ABCI 2.0 Forum Application** is a place where users can come and post messages in a forum running on a
+blockchain powered by [CometBFT](https://github.com/cometbft/cometbft) state machine replication engine
+
+- **Users**
+
+   - Can post messages (by submitting transactions)
+   - Can view all the message history (querying the blockchain)
+   - Banned if post messages contained curse words (curse words are tracked with vote extensions)
+
+## ABCI 2.0
+
+**This application demonstrates the use of various [ABCI 2.0](https://docs.cometbft.com/v1.0/spec/abci/) methods such as:**
+
+- PrepareProposal
+- ProcessProposal
+- FinalizeBlock
+- ExtendVote
+- VerifyVoteExtension
+- Commit
+
+
+- CheckTx
+- Query
+
```

Comment:
My understanding is that this folder contains the code (so, the _solution_) of the tutorial that actually starts in `docs/tutorials/forum-application/1.abci-intro.md`. If I'm getting if right, I think we should point anyone reading this README file to the start of the _actual_ tutorial.